test/compress/issue-1202.js

```diff
@@ -7,18 +7,16 @@ mangle_keep_fnames_false: {
         keep_fnames : false,
     }
     input: {
-        "use strict";
         function total() {
             return function n(a, b, c) {
                 return a + b + c;
             };
         }
     }
     expect: {
-        "use strict";
         function total() {
-            return function t(n, r, u) {
-                return n + r + u;
+            return function n(t, r, u) {
+                return t + r + u;
```

Comment:
The letter frequency affects the mangled function and variable names which were specifically chosen for this test.